src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -705,7 +706,8 @@ async Task<bool> createGlobalConfigFilesAsync()
 
                     var assemblyName = Path.GetFileNameWithoutExtension(assembly);
                     var shippedFile = Path.Combine(assemblyDir, "AnalyzerReleases", assemblyName, ReleaseTrackingHelper.ShippedFileName);
-                    if (File.Exists(shippedFile))
+                    var unshippedFile = Path.Combine(assemblyDir, "AnalyzerReleases", assemblyName, ReleaseTrackingHelper.UnshippedFileName);
+                    if (File.Exists(shippedFile) && File.Exists(unshippedFile))
```

Comment:
Should we assert that either both exists, or both don't (ie, assert that not only one of them is found)?